frontend/src/hooks/useResourceAttribute/utils.ts

```diff
@@ -93,6 +93,22 @@ export const resourceAttributesToTagFilterItems = (
 		value: `${res.tagValue}`.split(','),
 	}));
 };
+/* Convert resource attributes to trace filters items for queryBuilder */
+export const resourceAttributesToTracesFilterItems = (
+	queries: IResourceAttribute[],
+): TagFilterItem[] =>
+	queries.map((res) => ({
+		id: `${res.id}`,
+		key: {
+			key: convertMetricKeyToTrace(res.tagKey),
+			isColumn: false,
+			type: MetricsType.Resource,
+			dataType: DataTypes.String,
+			id: `${convertMetricKeyToTrace(res.tagKey)}--string--resource--true`,
+		},
+		op: `${res.operator}`,
+		value: `${res.tagValue}`.split(','),
```

Comment:
What happens if there is a `,` in the value of the filter?

Reply:
@srikanthccv,

here the `res.tagValue` comes as an array. 

I got the snippet (`value: `${res.tagValue}`.split(','),`) from this [line](https://github.com/SigNoz/signoz/blob/b96b44444fa9009b49e13d57527c1a4a12676688/frontend/src/hooks/useResourceAttribute/utils.ts#L110), so I thought it would be necessary based on the format of filter in QB. 

When I tested now, we get the same result even if we don't have `.split(',')`. 

Should I remove it?

Reply:
@srikanthccv, it's going to cause unintended behavior i.e. splitting the filter value. 

I had gotten it from this [line](https://github.com/SigNoz/signoz/blob/b96b44444fa9009b49e13d57527c1a4a12676688/frontend/src/hooks/useResourceAttribute/utils.ts#L93). I thought it would be necessary for the format of QB. but I tested by removing the `.split(',')`, and it works fine.

Removed it.